### DISCUSSION TITLE
[React] useEffect 공식문서 읽기(2)

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#588] [1. 개발 관련 글 모음]
각잡고 공식문서를 읽고 있으니 확실히 배우는 게 많네요...
[게시글 링크](https://amaran-th.github.io/React/useEffect%20%EC%A0%9C%EB%8C%80%EB%A1%9C%20%EC%9D%B4%ED%95%B4%ED%95%98%EA%B8%B0(2)/)

**Comment by shkisme** [comment]
오 `useEffect` 이거 Next.js 블로그 개발하면서 많이 사용하고 있는데 뭔지 잘 모르겠더라구용...
아마란스 블로그 글 많이 참고하겠습니다 🤩
